sdk/storage/azure-storage-blobs/test/perf/inc/azure/storage/blobs/test/download_blob.hpp

```diff
@@ -25,6 +25,8 @@ namespace Azure { namespace Storage { namespace Blobs { namespace Test {
    *
    */
   class DownloadBlob : public Azure::Storage::Blobs::Test::BlobsTest {
+  protected:
+    std::unique_ptr<std::vector<uint8_t>> m_downloadBuffer;
```

Comment:
If the buffer is to be used only inside the test, we don't need to make a member field of it, It can be a simple instance of std::vector created in the test